files/install_graphdb.sh

```diff
@@ -9,13 +9,13 @@ done
 
 timedatectl set-timezone UTC
 
-# Shred authorized_keys
-shred -u /root/.ssh/authorized_keys /home/ubuntu/.ssh/authorized_keys || true
-
 # Install Tools
 apt-get -o DPkg::Lock::Timeout=300 update -y
 apt-get -o DPkg::Lock::Timeout=300 install -y bash-completion jq nvme-cli openjdk-11-jdk unzip
 
+# Install Azure CLI
+curl -sL https://aka.ms/InstallAzureCLIDeb | bash
```

Comment:
As far as I remember the CLI has to be configured for a specific account, would this be done in the terraform or ?

Reply:
In the user data script, the procedure (AFAIK) is to use `az login --identity`, see https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-managed-identity